src/components/SubscriptionTable/__tests__/SubscriptionTable.test.tsx

```diff
@@ -140,27 +154,28 @@ describe('Subscription Table', () => {
       })
     ]);
 
-    it('pages', () => {
-      const { container } = render(<Table subscriptions={get('data')} />);
-      expect(container).toMatchSnapshot();
-    });
-
```

Comment:
Are we getting rid of this test? I feel like it would be just like the one below but no click, and assert the length is 10